src/components/Button.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Link from "next/link";
+
+const Button = (link, text, color) => {
+  return (
+    <Link href={link}>
+      <button style={{ backgroundColor: color }}>{text}</button>
```

Comment:
when you use `style` tag it is just regular CSS
but we are using tailwindCSS for this project, so use `className=""` instead
specifically you want to do `className={`${color}`}`

Also it looks like you didn't finish styling the button, like you didn't round the corners, you didn't put a width on it either, you will also need some padding